sherpa/csrc/offline-nemo-enc-dec-ctc-model-bpe.cc

```diff
@@ -46,4 +38,17 @@ torch::Tensor OfflineNeMoEncDecCTCModelBPE::GetLogSoftmaxOutLength(
   return {};
 }
 
+void OfflineNeMoEncDecCTCModelBPE::WarmUp(torch::Tensor features,
+                                          torch::Tensor features_length) {
+  // For Citrinet, the subsampling_factor_ is 16.
+  // For Conformer CTC, the subsampling_factor_ is 4.
+  auto ivalue = Forward(features, features_length);
+  auto log_prob = GetLogSoftmaxOut(ivalue);
+
+  vocab_size_ = log_prob.size(-1);
+  subsampling_factor_ =
+      (features_length.cpu().to(torch::kInt).item<int32_t>() + 15) /
```

Comment:
Should be +7 - at least for now we dont train CTC models with more than 8x stride

Reply:
Thanks. Fixed in https://github.com/k2-fsa/sherpa/pull/338